src/stash/protos/objects_v19.proto

```diff
@@ -334,6 +334,37 @@ message TimestampAntichain {
     repeated Timestamp elements = 1;
 }
 
+enum ObjectType {
```

Comment:
What is the scope of this not having existed before? It'd only be problem during upgrade but is fine now because it's unused?

Reply:
> It'd only be problem during upgrade but is fine now because it's unused

Exactly! Theoretically we could require snapshots to only be created when migrations are written, but that might get a little confusing. For example, a no-op migration wouldn't need to snapshot the protos, but then you'd need to make two snapshots when you have a migration that does something.

IMO always requiring a snapshot when the version is bumped is easier to reason about and enforce